proto/qdrant/cloud/cluster/backup/v1/backup.proto

```diff
@@ -389,6 +390,40 @@ message Backup {
   // The identifier of the backup schedule (in GUID format).
   // This is a read-only field and will be present if the backup was created by a backup schedule.
   optional string backup_schedule_id = 9 [(buf.validate.field).string = {uuid: true}];
+  // Cluster details associated with the backup.
+  // Identity fields reflect the latest cluster state; configuration reflects the state at backup time.
+  // This is a read-only field and will be available after the backup is created.
+  ClusterInfo cluster_info = 12;
+}
+
+// Represents the cluster details associated with a backup.
+// The identity fields (name, cloud_provider_id, cloud_provider_region_id) reflect the latest cluster state.
+// The configuration field is immutable and represents the values at backup time.
+message ClusterInfo {
+  // The latest name of the cluster that this backup was token from.
+  string name = 1 [(buf.validate.field).string = {
```

Comment:
@Robert-Stam @mikheillomidzeq the requirements were that the name should be the latest name of the cluster, while the configuration should be the one at the time the backup was created. I think this behavior can be slightly confusing, and I tried adding comments to make it as clear as possible.